nvflare/private/fed/client/client_executor.py

```diff
@@ -215,6 +215,12 @@ def start_app(
         )
         thread.start()
 
+    def notify_job_status(self, job_id, job_status):
```

Comment:
When client engine calls this, the job_status can be different.  However, on the engine side, the message is always 'started.'  Maybe we need the sync this.

And if the run_process from self.run_processes.get(job_id) is an empty dict, the job_status is not updated.

Reply:
The self.run_processes keep the accurate job status during the client job life cycle. the "check_status" will report from this. The life cycle is "starting" --> "started" --> "stopped". 

When self.run_processes.get(job_id) is an empty dict, it means no job running from this job. "check_status" command will report properly.